contracts/vaults/Deposit.sol

```diff
@@ -97,6 +97,7 @@ contract Deposit is Base {
         // if the user already has funds in the pool, give the previous reward
         if (_user.shares > 0) {
             uint256 pending = _user.shares * poolInfos[_pid].rewardPerShare / 1e12 - _user.rewardDebt;
+            userInfo[_pid][msg.sender].rewardDebt += pending;
```

Comment:
pls let's not change this in this pr?

Reply:
Fixed, sorry must have left it there while working on the other stuff